### PR TITLE
libmaxminddb: add version 1.10.0

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **libmaxmind/1.10.0**

https://github.com/maxmind/libmaxminddb/compare/1.9.1...1.10.0

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by RubenRBS — DISMISSED** [review]
Thanks (Pushed minor changes to retrigegr the PR!)

**Comment by toge** [comment]
@RubenRBS 
Thanks for your improvement!

File: recipes/libmaxminddb/all/conanfile.py [hunk 1]
```diff
@@ -81,7 +80,6 @@ def package_info(self):
 
         if self.settings.os != "Windows":
```
Comment (RubenRBS) [inline recipes/libmaxminddb/all/conanfile.py:81]: Should this line include checking if `self.options.get_safe("with_binaries")` is true?

**Review by RubenRBS — COMMENTED** [review]
Minor comment, otherwise looks good to me :)

File: recipes/libmaxminddb/all/conanfile.py (hunk 1 above)
Comment (toge) [reply recipes/libmaxminddb/all/conanfile.py]: @RubenRBS 
Thank you for your comment.
I fixed it.

**Review by danimtb — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]
Thanks:)

**Pull request merged**